app/_redirects

```diff
@@ -39,6 +39,11 @@ https://prometheus.metrics.kuma.io/path /docs/LATEST_RELEASE/reference/kubernete
 https://traffic.kuma.io/exclude-inbound-ports /docs/LATEST_RELEASE/reference/kubernetes-annotations/#traffic-kuma-io-exclude-inbound-ports/ 302
 https://traffic.kuma.io/exclude-outbound-ports /docs/LATEST_RELEASE/reference/kubernetes-annotations/#traffic-kuma-io-exclude-outbound-ports/ 302
 
+# Argo CD section redirects post submodule
+
+/docs/2.3.x/installation/helm/#argo-cd  /docs/2.3.x/production/cp-deployment/kubernetes/#argo-cd    
+/docs/2.2.x/installation/helm/#argo-cd  /docs/2.2.x/production/cp-deployment/kubernetes/#argo-cd
```

Comment:
I'm not sure why this redirect only applies for 2.2 and 2.3. Seems to me that we should add to _common_redirect a redirect:

```
/docs/:version/installation/helm /docs/:version/production/cp-deployment/kubernetes/#helm
```

WDYT?

Reply:
@lahabana Ah, good point. I did it this way because the [installation/helm](https://kuma.io/docs/2.1.x/installation/helm/) page does exist in 2.1.x or earlier, so otherwise it would be a 404 if it tried to go to /docs/2.1.x/production/cp-deployment/kubernetes/#helm.